### PR TITLE
chore(scan): remove adjudication

### PR DESCRIPTION
**Pull request by eventualbuddha**
## Overview

We used to have adjudication features in VxCentralScan, i.e. for adjudicating overvotes/undervotes/etc. We took the UI for that out a while back but never fully removed the backend implementation. Since this is unlikely to ever be a product requirement, or if it is it won't look like this, it's better to just remove it.

## Demo Video or Screenshot
n/a

## Testing Plan
Automated testing.

## Checklist
- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [ ] I have added JSDoc comments to any newly introduced exports

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: frontends/bsd/src/screens/ballot_eject_screen.tsx [hunk 1]
```diff
@@ -153,63 +146,59 @@ export function BallotEjectScreen({
...
       }
     }
-  }, [reviewInfo, logger, userRole]);
+  }
 
   useEffect(() => {
     void (async () => {
       const frontAdjudicationComplete =
-        !!frontMarkAdjudications ||
+        !!isFrontAdjudicationDone ||
```
Comment (jonahkagan) [inline frontends/bsd/src/screens/ballot_eject_screen.tsx:198]: I think the `!!` here is unnecessary now

File: services/scan/schema.sql [hunk 2]
```diff
@@ -35,6 +35,7 @@ create table sheets (
   -- When adjudication is finished, these values are updated to now.
   front_finished_adjudication_at datetime,
   back_finished_adjudication_at datetime,
+  finished_adjudication_at datetime,
```
Comment (jonahkagan) [inline services/scan/schema.sql:30]: I'm a little confused - why is there still a concept of adjudication if we removed the feature?

**Review by jonahkagan — APPROVED** [review]

File: services/scan/schema.sql (hunk 2 above)
Comment (eventualbuddha) [reply services/scan/schema.sql:30]: We still have a concept of “this ballot has a problem” like an over vote that the user can decide something about (accept or reject). So we need this value to indicate when it was accepted. I'm happy to update the language if you can think of something better.

Comment (jonahkagan) [reply services/scan/schema.sql:30]: Ah ok, I think that still sounds like adjudication to me! And [NIST](https://pages.nist.gov/ElectionGlossary/#:~:text=adjudication,ballots%20to%20reflect%20voter%20intent.) agrees:
Screen Shot 2022-11-03 at 11 17 35 AM

Maybe I was just thrown off by the PR title

**Pull request merged**